base/xx-cargo

```diff
@@ -52,9 +53,9 @@ if [ ! -f "$done_file" ]; then
     if [ -n "$rustup" ]; then
       rustup target add "$(xx-info)" 2>/dev/null >/dev/null
     elif [ -f /etc/alpine-release ]; then
-      xx-apk add rust-stdlib 2>/dev/null >/dev/null
+      XX_VENDOR=$vendor xx-apk add rust-stdlib 2>/dev/null >/dev/null
```

Comment:
Can you explain this and add comments. This flipping between `XX_VENDOR` and `vendor` on different conditions is hard to reason about.